CHANGELOG.asciidoc

```diff
@@ -29,6 +29,17 @@ See Conventional Commits (https://conventionalcommits.org) for commit guidelines
 [[release-notes-5.x]]
 === RUM JS Agent version 5.x
 
+[[release-notes-5.10.0]]
+==== 5.10.0 (2021-12-07)
+[float]
+===== Features
+* Make server prefix URL used to make requests to the APM Server configurable: {issue}1078[#1078]
```

Comment:
So in release notes, its good to eloborate on a specific feature than keeping it short. I would write this as 

```suggestion
* Introduce new config option `serverUrlPrefix` to make the APM server URL configurable: {issue}1078[#1078]
```